content/en/docs/concepts/services-networking/service.md

```diff
@@ -241,9 +241,18 @@ There are a few reasons for using proxying for Services:
   on the DNS records could impose a high load on DNS that then becomes
   difficult to manage.
 
+Below we'll describe how various kube-proxy implementations work.  In general
+you should note that, when running `kube-proxy`, kernel level rules may be
+modified (for example, iptables rules might get created), which won't get cleaned up, 
```

Comment:
Hi @jayunit100 . Some feedback:
What do you think about removing the first sentence and possibly place this paragraph (or part of the paragraph) in a Note shortcode to callout the content?

Line 247, extra spaces before "Thus"
Line 249, fix typo, "low level", privileged network ...
Line 249, extra spaces before "Although"

Reply:
sure, commit a suggestion?